tests/lib/rules/indent.js

```diff
@@ -7693,6 +7757,47 @@ ruleTester.run("indent", rule, {
             `,
             errors: expectedErrors([5, 4, 8, "Identifier"])
         },
+        {
+            code: unIndent`
+              condition
+              ? () => {
+              return true
+              }
+              : condition2
+              ? () => {
+              return true
+              }
+              : () => {
+              return false
+              }
+            `,
+            output: unIndent`
+              condition
+                ? () => {
+                    return true
+                  }
+                : condition2
+                  ? () => {
+                      return true
+                    }
+                  : () => {
+                      return false
+                    }
+            `,
+            options: [2, { offsetTernaryExpressions: true }],
```

Comment:
I'd love to have both valid and invalid tests with `offsetTernaryExpressions: false` as a guard against regressions of the default value.